<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="minimum_label">
     <property name="text">
      <string>Minimum:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="maximum_label">
     <property name="text">
      <string>Maximum:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QComboBox" name="ties_comboBox"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="tie2_label">
     <property name="text">
      <string>Tie to:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="minimum_lineEdit"/>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="maximum_lineEdit"/>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="fixed_radioButton">
        <property name="text">
         <string>Fixed</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fit_radioButton">
        <property name="text">
         <string>Fit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="unchanged_radioButton">
        <property name="text">
         <string>Unchanged</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="value_label">
     <property name="text">
      <string>Value:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="value_lineEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>value_lineEdit</tabstop>
  <tabstop>fixed_radioButton</tabstop>
  <tabstop>fit_radioButton</tabstop>
  <tabstop>unchanged_radioButton</tabstop>
  <tabstop>minimum_lineEdit</tabstop>
  <tabstop>maximum_lineEdit</tabstop>
  <tabstop>ties_comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
